docs/src/Modal.doc.js

```diff
@@ -23,85 +32,497 @@ card(
         type: 'string',
         required: true,
         description:
-          'String that clients such as VoiceOver will read to describe the modal. Always localize the label.',
-        href: 'accessibility',
+          'String that clients such as VoiceOver will read to describe the modal. Always localize the label. See the [Accessibility section](#Accessibility) for more info.',
       },
       {
         name: 'align',
         type: `"center" | "left"`,
         defaultValue: 'center',
-        description: 'Use to specify the alignment of `heading` & `subHeading` strings',
-        href: 'headingOptions',
+        description:
+          'Use to specify the alignment of `heading` & `subHeading` strings. See the [Heading variant](#Heading) for more info.',
       },
       {
         name: 'children',
         type: 'React.Node',
+        description:
+          "Supply the element(s) that will be used as Modal's main content. See the [Best Practices](#Best-practices) for more info.",
       },
       {
         name: 'closeOnOutsideClick',
         type: 'boolean',
-        description: 'Close the modal when you click outside of it',
+        description:
+          'Close the modal when you click outside of it. See the [outside click variant](#Preventing-close-on-outside-click) for more info.',
         defaultValue: true,
-        href: 'closeOnOutsideClickExample',
       },
       {
         name: 'footer',
         type: 'React.Node',
-        href: 'sizesExample',
+        description:
+          "Supply the container element(s) or render prop that will be used as Modal's custom footer. See the [Best Practices](#Best-practices) for more info.",
```

Comment:
I don't believe `footer` supports function children. It only accepts a Node, not `() => Node`